Cargo.toml

```diff
@@ -23,6 +23,8 @@ bytes = "1"
 mime = "0.3.14"
 sha1 = "0.10"
 httpdate = "1"
+rand = { version = "0.8", optional = true }
```

Comment:
I don't think I want to depend on `rand` in this crate, or choose what RNG is used for people. If we added just a `TryFrom` implementation, people can do this themselves.

Reply:
Changed to construct from bytes.